static/src/javascripts/projects/commercial/modules/hosted/youtube.ts

```diff
@@ -90,5 +90,5 @@ export const initHostedYoutube = (el: HTMLElement): void => {
 			},
 		},
 		el.dataset.assetId as string,
-	);
+	).then(() => void 0);
```

Comment:
`initYoutubePlayer` returns a `YT.Player`, so we need to return `void` instead to keep this signature similar.